Ruddarr/Views/Movies/MovieForm.swift

```diff
@@ -66,6 +68,21 @@ struct MovieForm: View {
         .tint(.secondary)
     }
 
+    var tagsField: some View {
+        NavigationLink {
+            TagList(selected: $tags)
+                .onChange(of: tags) {
+                    movie.tags = Array(tags)
+                }
```

Comment:
I dropped storing default tags, I feel like this would get annoying. We can restore that if people ask for it.